File: php-laravel/lib/app/Models/RequestTimeFilterPostcodeSectorsArrivalSearch.php
<?php
/**
 * RequestTimeFilterPostcodeSectorsArrivalSearch
 */
namespace app\Models;

/**
 * RequestTimeFilterPostcodeSectorsArrivalSearch
 */
class RequestTimeFilterPostcodeSectorsArrivalSearch {

    /** @var string $id */
    private $id;

    /** @var \app\Models\RequestTransportation $transportation */
    private $transportation;

    /** @var int $travelTime */
    private $travelTime;

    /** @var \DateTime $arrivalTime */
    private $arrivalTime;

    /** @var double $reachablePostcodesThreshold */
    private $reachablePostcodesThreshold;

    /** @var \app\Models\RequestTimeFilterPostcodeSectorsProperty[] $properties */
    private $properties;

    /** @var \app\Models\RequestRangeFull $range */
    private $range;

}

File: fsharp-giraffe-server/traveltimeplatform/src/model/RequestTimeFilterPostcodeSectorsArrivalSearch.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.RequestRangeFull
open traveltimeplatform.Model.RequestTimeFilterPostcodeSectorsProperty
open traveltimeplatform.Model.RequestTransportation

module RequestTimeFilterPostcodeSectorsArrivalSearch = 

  //#region RequestTimeFilterPostcodeSectorsArrivalSearch


  type RequestTimeFilterPostcodeSectorsArrivalSearch = {
    Id : string;
    Transportation : RequestTransportation;
    TravelTime : int;
    ArrivalTime : DateTime;
    ReachablePostcodesThreshold : double;
    Properties : RequestTimeFilterPostcodeSectorsProperty[];
    Range : RequestRangeFull;
  }
  //#endregion
  